docs/trouble-shooting.md

```diff
@@ -6,6 +6,7 @@
 
 <!--
     Copyright (c) 2015, Joyent, Inc.
+    Copyright 2024 MNX Cloud, Inc.
```

Comment:
You'll have to bump that to 2025 now.  :(

Reply:
Done